CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ find_package(Boost 1.54.0 COMPONENTS system regex thread REQUIRED)
 include_directories(${Boost_INCLUDE_DIR})
 
 if(APPLE)
-  set(OPENSSL_ROOT_DIR "/usr/local/opt/openssl")
```

Comment:
Unless OpenSSL has been removed entirely from OS X as base library, that is, openssl is now installed to /usr/local instead of /usr/local/opt in 10.11?